automl/batch_predict.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package automl contains samples for Google Cloud AutoML API.
+package automl
+
+// [START automl_batch_predict]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	automl "cloud.google.com/go/automl/apiv1beta1"
+	automlpb "google.golang.org/genproto/googleapis/cloud/automl/v1beta1"
+)
+
+// batchPredict does a batch prediction.
+func batchPredict(w io.Writer, projectID string, location string, modelID string, inputURI string, outputURI string) error {
```

Comment:
Can we add a test for this sample?

Reply:
@davidcavazos , are you working on the tests for the rest of these samples?

Reply:
I was working on them but paused them due to our discussion that we shouldn't be creating datasets and/or models on all of the testing projects.